File: AirportManagement.API/AirportManagement.API/Data/Models/FlightManifest.cs
using System;
namespace AirportManagement.API.Models
{
	public class FlightManifest
	{
		public int Id { get; set; }

		public DateTime CreatedAt { get; set; }

		public DateTime UpdatedAt { get; set; }

		public int FlightId { get; set; }

		public Flights? Flights { get; set; }

		public int BookingId { get; set; }

		public Booking? Booking { get; set; }
	}
}

File: AirportManagement.SPA/src/app/_models/FlightManifest.ts
import { Booking } from "./Booking";
import { Flight } from "./Flight";

export interface FlightManifest {
    Id: number;
    CreatedAt: Date;
    UpdatedAt: Date;
    FlightId: number;
    Flights: Flight;
    BookingId: number;
    Booking: Booking;
}
